modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -0,0 +1,22 @@
+= Connected Apps
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can connect applications to Anypoint Platform without granting them unrestricted access to data or having to share sensitive credentials.
```

Comment:
I think our style guide says to use app instead of application.

Reply:
Change made throughout.